### DISCUSSION TITLE
更换gravatar头像源后无法识别qq头像

### DISCUSSION THREAD

**Discussion by zbzzh** [ArtalkJS/Artalk#730] [Q&A]
能否设置如果是QQ邮箱的话直接走QQ头像源，不走gravatar，这两者是不同的

**Comment by qwqcode** [comment]
cravatar 符合你的需求吗？https://cravatar.com/developer/api

头像镜像地址填 `https://cravatar.cn/avatar/`

**Reply by zbzzh** [reply]
这个第一次加载非常慢，几乎需要10s，建议还是程序原生支持qq头像
